External/SDK/Clang/Linux/Clang_CI.bff

```diff
@@ -0,0 +1,63 @@
+// Clang
```

Comment:
Seems pretty reasonable to bring these *_CI files back for Linux and OSX until they can be merged so they auto-detect like the Windows bffs do now.